cmd/grpcurl/grpcurl.go

```diff
@@ -52,13 +52,20 @@ var (
 	key = flags.String("key", "", prettify(`
 		File containing client private key, to present to the server. Not valid
 		with -plaintext option. Must also provide -cert option.`))
-	protoset    multiString
-	protoFiles  multiString
-	importPaths multiString
-	addlHeaders multiString
-	rpcHeaders  multiString
-	reflHeaders multiString
-	authority   = flags.String("authority", "", prettify(`
+	protoset      multiString
+	protoFiles    multiString
+	importPaths   multiString
+	addlHeaders   multiString
+	rpcHeaders    multiString
+	reflHeaders   multiString
+	expandHeaders = flags.Bool("expand-headers", false, prettify(`
+		If set any environmental variables contained contained in any additional, rpc,
```

Comment:
I think some of the wording in here is a little awkward. How about something like this instead:

```
If set, headers may use '${NAME}' syntax to reference environment variables. These
will be expanded to the actual environment variable value before sending to the server.
For example, if there is an environment variable defined like FOO=bar, then a header of
'key: ${FOO}' would expand to 'key: bar'. This applies to -H, -rpc-header, and
-reflect-header options. No other expansion/escaping is performed. This can be used
to supply credentials/secrets without having to put them in command-line arguments.
```